test/multiSamlStrategy.spec.ts

```diff
@@ -25,6 +25,18 @@ describe("MultiSamlStrategy()", function () {
     expect(strategy).to.be.an.instanceOf(Strategy);
   });
 
+  it("does not require issuer in the SamlOptionsCallback", function () {
```

Comment:
This is an inadequate test. It simply defines a function; it never uses it. With this test we have no way of knowing if this function is fit for purposes.